src/chapter1/isbn.hs

```diff
@@ -0,0 +1,35 @@
+module Chapter1.Isbn where
+import Data.List
+import Data.Char
+rmdash :: String -> String
+rmdash x = delete '-' (delete '-' (delete '-' x))
+
+cformat :: String -> String
```

Comment:
why is this function returning a blank string? can you think of a more "type-safe" way of expressing what you are doing?